cirq/google/optimizers/convert_to_xmon_gates.py

```diff
@@ -65,6 +64,19 @@ def _convert_one(self, op: 'cirq.Operation') -> 'cirq.OP_TREE':
 
         return NotImplemented
 
+    def _is_native_xmon_op(self, op: 'cirq.Operation') -> bool:
+        """Check if the gate within an operation is a native xmon gate.
+
+        Args:
+            op: Input operation.
+
+        Returns:
+            True if the operation is native to the xmon, false otherwise.
+        """
+        return (isinstance(op, ops.GateOperation) and isinstance(
```

Comment:
Does this differ from the XMON gateset? https://github.com/quantumlib/Cirq/blob/059006d951541d399bbd8fba0f3eb43553dcc18b/cirq/google/gate_sets.py#L92

If it doesn't, can we reference that as the source of truth for the supported gates?

Reply:
Done.  Had to import within the function to avoid circular deps, but otherwise should be okay.

Reply:
Note: it actually needs to reference cirq.google.XmonDevice (the device) not cirq.google.XMON (the gateset) since those two are actually slightly out of sync, due to PhasedXZ.